frontend/src/layouts/NetworkDropdown.vue

```diff
@@ -1,49 +0,0 @@
-<template>
```

Comment:
_(arbitrary comment location to start a thread)_

When connected on Optimism / other chains, it no longer shows my ENS name, but I think it should based on how resolution is supposed to work, cc @apbendi to confirm

<img width="339" alt="image" src="https://user-images.githubusercontent.com/17163988/197590469-f7564212-c44d-44d7-9f32-6737de9cc7e7.png">

Reply:
Yeah, ENS reverse resolution should always use mainnet. I believe we provided our own Provider to do this. Perhaps Blocknative doesn't support this—is there a way to override?